packages/x-charts/src/Gauge/GaugeContainer.tsx

```diff
@@ -85,40 +84,38 @@ const GaugeContainer = React.forwardRef(function GaugeContainer(
 
   return (
     <SizeProvider width={inWidth} height={inHeight}>
-      <SvgRefProvider>
-        <DrawingAreaProvider margin={{ left: 10, right: 10, top: 10, bottom: 10, ...margin }}>
-          <GaugeProvider
-            value={value}
-            valueMin={valueMin}
-            valueMax={valueMax}
-            startAngle={startAngle}
-            endAngle={endAngle}
-            outerRadius={outerRadius}
-            innerRadius={innerRadius}
-            cornerRadius={cornerRadius}
-            cx={cx}
-            cy={cy}
+      <DrawingAreaProvider margin={{ left: 10, right: 10, top: 10, bottom: 10, ...margin }}>
+        <GaugeProvider
+          value={value}
+          valueMin={valueMin}
+          valueMax={valueMax}
+          startAngle={startAngle}
+          endAngle={endAngle}
+          outerRadius={outerRadius}
+          innerRadius={innerRadius}
+          cornerRadius={cornerRadius}
+          cx={cx}
+          cy={cy}
+        >
+          <ResizableContainer
+            role="meter"
+            aria-valuenow={value === null ? undefined : value}
+            aria-valuemin={valueMin}
+            aria-valuemax={valueMax}
+            {...other}
           >
```

Comment:
Why keeping this component?

If it's only for the aria properties, you can extend `ChartsSurfaceProps` with `extends React.SVGProps<SVGSVGElement>`. To keep the `aria-hidden` we can add a `<g/>`

Reply:
I forgot about it... (: